### ISSUE TITLE
History Viewer - edit form overflow issue

### ISSUE THREAD

**Issue by blueo** [silverstripe/silverstripe-versioned-admin#125]
Hi there,

I've got a vanilla 4.4.2 install (via silverstripe/recipe-cms) and this history viewer is causing odd scroll behaviour if the the edit form is long.

for example, if I add some fields to getCMSFields to create a long form eg:
```
public function getCMSFields()
        {
            $fields = parent::getCMSFields();
            $fields->addFieldsToTab('Root.Main', [
                new UploadField('Test', 'Test'),
                new UploadField('Test1', 'Test'),
                new UploadField('Test2', 'Test'),
                new UploadField('Test3', 'Test'),
                new UploadField('Test4', 'Test'),
                new UploadField('Test5', 'Test'),
                new UploadField('Test6', 'Test'),
                new UploadField('Test7', 'Test')
            ]);

            return $fields;
        }
```

then publish a page to get a history record. On viewing a history record i get this situation:
![Screen Shot 2019-08-13 at 2 43 35 PM](https://user-images.githubusercontent.com/948122/62911749-995be480-bdd9-11e9-89e7-ca4a22c593ec.png)

basically the edit form breaks below the bottom of the viewport. I have tried adding a overflow-hidden to the cms container as per: https://github.com/silverstripe/silverstripe-admin/pull/865 but this means the history viewer is not scrollable.

manually adding `overflow: auto` to the `.history-viewer` class allows the edit form to scroll (and the preview pane next to it).

**Comment by blueo** [comment]
https://github.com/silverstripe/silverstripe-versioned-admin/pull/126

**Comment by Cheddam** [comment]
@blueo Assuming the PR you raised fixes this issue, I'm closing this out 👍Thanks!

**Comment by blueo** [comment]
oops yep it did indeed thanks @Cheddam
